src/workflows-app/components/WorkflowsAppNavPanel.ts

```diff
@@ -76,23 +76,24 @@ type WorkflowsAppNavPanelProps = {
   launching: boolean;
   createWorkflowsApp: Function;
   pageReady: boolean;
-  setLoading: Function;
-  signal: Function;
+  setLoading: (value: boolean) => void;
+  signal?: AbortSignal;
```

Comment:
made signal optional to mirror the ajax arg it is used for also being optional (and ease caller burden in tests)